v8/prettybasic/templates/annotation_helper.tmpl

```diff
@@ -0,0 +1,16 @@
+{% macro css() %}
```

Comment:
After you undo the `base.tmpl` → `prettybasic.tmpl` rename, please compare your template files to the ones in `base-jinja`, and if the files are identical, remove your copy so that the copy in `base-jinja` is inherited.

Reply:
Do I really need to remove those duplicate templates? I would like to keep them because I still work on the them and I would like to keep everything in place to improve my understanding of Jinja2. I mean, if it leads to problems I would remove them for this repo and keep them in my dev environment, but it would save some time.

Reply:
Yes, you do. Feel free to keep them in a local copy, or copy them again from base-jinja (with `nikola theme -c`) when you do make changes to them, and feel free to override template files if you want to change their appearance. But by copying templates you didn’t modify, you make any changes to Nikola itself harder. Some of the templates you copied are effectively Nikola-internal and themes on the index should never copy them (like comment helpers). For the rest, we would have to edit every theme when we make some changes to the more “general” templates – for example, many themes don’t edit `gallery.tmpl`, because our implementation fits them well, and it lets us improve the gallery code for more themes with just one change in core.

Reply:
Got it, thank you for this explanation.